tests/Hedgehog.Tests/PropertyTests.fs

```diff
@@ -0,0 +1,21 @@
+﻿module Hedgehog.Tests.PropertyTests
+
+open Hedgehog
+open Expecto
+open TestDsl
+
+let propertyTests = testList "Property tests" [
+    fableIgnore "generated C# list of five elements is not abbreviated in the failure report" <| fun _ ->
+        Expect.throwsC
```

Comment:
Instead of asserting that some code throws an exception, I think it would be better to test if the rendered property doesn't contain the substring `"..."`.  See [here for a C# example](https://github.com/hedgehogqa/fsharp-hedgehog/pull/318/files#diff-c5321d72bf30f64c2b6fb902119ac3bdf4b9c9720755247f774ff7c13c0b4208R34-R37) of how to do that.

Reply:
Done